docs/apm/api.asciidoc

```diff
@@ -9,14 +9,19 @@
 Some APM app features are provided via a REST API:
 
 * <<agent-config-api>>
+* <<annotation-api>>
 
 TIP: Kibana provides additional <<api,REST APIs>>,
 and general information on <<using-api,how to use APIs>>.
 
+// Basic auth or API can be used
+// Need to talk about permissions req'd
```

Comment:
> When using the APM API, the user needs Kibana write permissions for APM (similar to the agent configuration APIs IIRC)

Can you be more specific? My first thought was `create_doc` on `apm-*` indices and `create_index` on `apm-*` indices, as described [here](https://www.elastic.co/guide/en/apm/server/current/feature-roles.html#privileges-to-publish-events), but that doesn't work. Presumably because these annotations are not stored in an index that matches that pattern.

Reply:
Yeah, the user indexing the annotations needs those permissions for `observability-annotations`, or whatever index name they set for `xpack.observability.annotations.index`. Additionally, for the APM endpoint, they need feature privileges (read and write) for APM: https://www.elastic.co/guide/en/kibana/current/kibana-privileges.html.

Reply:
Here are the minimum privileges I was able to get to work:

Feature: `APM` --> `all`
Index: `observability-annotations`. Privileges: `create_index`, `create_doc`, `manage`, and `read`.

Does this sound correct to you? This is different from what you described, but I get 500s without `manage` and `read` on `observability-annotations`.

Reply:
You're right - when we index an annotation we read it from ES immediately after so we can return the annotation as it is stored in ES. `manage` is needed for checking if the index exists (`view_index_metadata` works as well).